packages/apps/human-app/frontend/src/shared/helpers/regex.ts

```diff
@@ -3,9 +3,8 @@ export const password8Chars =
 export const passwordUppercase = /(?=.*[A-Z])/;
 export const passwordLowercase = /(?=.*[a-z])/;
 export const passwordNumeric = /(?=.*[0-9])/;
-export const passwordSpecialCharacter =
-  /(?=.*[$^*.[\]{}()?"!@#%&/\\,><':;|_~`=+-])/;
+export const passwordSpecialCharacter = /[!@#$%^&+=*]/; // allowed characters: @#$%^&+=!*]
```

Comment:
I think we don't need this since the only restriction we'll have for the password is length.

Reply:
done